hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/NodeExtensionCommon.java

```diff
@@ -79,6 +79,7 @@ void handlePacket(Packet packet) {
     }
 
     void printNodeInfo(ILogger log, String addToProductName) {
+        log.info("Cluster name: " + node.getConfig().getClusterName());
```

Comment:
This makes it print before the message "Hazelcast Jet <version> starting", that line should come first I think.

Reply:
@mtopolnik, we have a fine level logging for "... Hazelcast Serialization..." right after "Hazelcast Jet starting",  should I place after there?

Reply:
I think it will be fine below the `Hazelcast Jet starting` line since the `fine` log level signals lower importance.

Reply:
Ok, I changed the order and also move the logging to a method like the other. :D